Fix the file name of the blog post and its title (#56)
Open-Science-Community-Saudi-Arabia/OSCSA-en-blog: posts/outreachy-intro/index.qmd

 I have an educational background in Petroleum Engineering and had a career change to Computer Science related field because i realize that technological fields offered more opportunities to use my creativity and I love building things!!! and I do hope to use my skills to contribute to the Oil and Gas industry.

### As an Outreachy Intern

> I am one of the successful candidates selected to participate in the [Outreachy 2023 Internship](https://www.outreachy.org/) , I applied to Outreachy because I see it as an opportunity to learn new technologies and contribute to varities of community through Open Source.  The Outreachy Internship is a three months program to contribute to any Open Source community of your choice. For the selection period, I chose [Open Science Community Saudi Arabia (OSCSA)](https://github.com/Open-Science-Community-Saudi-Arabia) because their goals were clearly stated.  I contributed to improving the Open Science Community Saudi Arabia (OSCSA) [website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website) and also collaborated in developing a [Massive Open Online Courses(MOOCs)](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs/tree/dev_team2) platform. The mentors were super helful and I enjoyed contributing to the community.  

Reviewer: ```suggestion
I am one of the successful candidates selected to participate in the [Outreachy 2023 Internship](https://www.outreachy.org/) , I applied to Outreachy because I see it as an opportunity to learn new technologies and contribute to varieties of communities through Open Source.  The Outreachy Internship is a three months program to contribute to any Open Source community of your choice. For the selection period, I chose [Open Science Community Saudi Arabia (OSCSA)](https://github.com/Open-Science-Community-Saudi-Arabia) because their goals were clearly stated.  I contributed to improving the Open Science Community Saudi Arabia (OSCSA) [website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website) and also collaborated in developing a [Massive Open Online Courses(MOOCs)](https://github.com/Open-Science-Community-Saudi-Arabia/MOOCs/tree/dev_team2) platform. The mentors were super helpful and I enjoyed contributing to the community.  
```

:::

::: {.g-col-lg-6 .g-col-12 }

> I am Richie Moluno, a Fullstack developer but more of backend development. My major stack is MongoDb ExpressJS and NodeJs. Currently I study Mechanical Engineering at the University of Benin, Benin City. I originally wanted to study Robotics because of the Iron-man movie. I really love robots, the process of converting computer codes to mechanical movements really intrigues me. When it was time to move into the University, the school i had in mind didn't have any course for Robotics. So based on my knowledge then, i felt Mechanical Engineering would be the closest alternative to Robotics. 

Reviewer: ```suggestion
I am Richie Moluno, a Fullstack developer but more of a backend developer. My major stack is MongoDb ExpressJS and NodeJs. Currently, I am studying Mechanical Engineering at the University of Benin, Benin City. I originally wanted to study Robotics because of the Iron-man movie. I really love robots, the process of converting computer codes to mechanical movements really intrigues me. When it was time to move to the University, the school I had in mind didn't have any courses in Robotics. So based on my knowledge then, I felt Mechanical Engineering would be the closest alternative to Robotics. 
```
